Makefile

```diff
@@ -409,6 +392,32 @@ ifneq ($(HAS_WEBAPP),)
 endif
 	rm -fr build/bin/
 
+.PHONY: logs
+logs:
```

Comment:
Suggestion from the AI to document the newly added targets, like logs, logs-watch and mock